core/management/commands/generate_test_data.py

```diff
@@ -15,31 +15,9 @@
 
 
 class Command(BaseCommand):
-    help = 'Closes the specified poll for voting'
+    help = 'Create some test data in the database.'
 
     def handle(self, *args, **options):
-        f = io.StringIO()
-        with contextlib.redirect_stdout(f):
-            call_command('sqlflush')
-        flush_db = f.getvalue()
-
-        with connection.cursor() as cursor:
-            statements = flush_db.split('\n')
-            for statement in statements:
-                if not statement:
-                    continue
-                cursor.execute(statement)
-
-        call_command('migrate')
-
-        # setting the seed so output is consistent
-        fake = Faker()
-        fake.seed(1)
```

Comment:
do we still want these? i'm not sure if there was a reason @jonathan-s designed this to have consistent output. having non-consistent output would test more cases, just not deterministically.

Reply:
Yes - oops. It's back in a rebased version.